### PR TITLE
Allow customization of tablist footer

### LINKED ISSUE #573: Add IP and server name to tab footer
OCN showed the IP and server name in the tab list footer.

image

With a few servers branching into more of a network than a single PGM instance this could be useful.

These vales should be config options and not [hardcoded](https://github.com/StratusNetwork/projectares/pull/26#issuecomment-319222932).

### PR DESCRIPTION
**Pull request by applenick**
# Custom Tablist Footer

Resolves #573 

New config section:
```yaml
tablist:
  left: ""
  right: ""

```
## Screenshots
Example of footer shown with additional stats
footer-match

Example of footer shown as an observer
footer-hello-world

Signed-off-by: applenick <applenick@users.noreply.github.com>

### CODE REVIEW

File: core/src/main/java/tc/oc/pgm/tablist/MatchFooterTabEntry.java [hunk 1]
```diff
@@ -60,13 +62,24 @@ public BaseComponent getContent(TabView view) {
...
+
     content
         .append(TranslatableComponent.of("match.info.time", TextColor.GRAY))
         .append(": ", TextColor.GRAY)
         .append(
             TimeUtils.formatDuration(match.getDuration()),
             this.match.isRunning() ? TextColor.GREEN : TextColor.GOLD);
 
+    if (rightContent != null) {
+      content.append(" - ", TextColor.DARK_GRAY).append(rightContent);
```
Comment (Pablete1234) [inline core/src/main/java/tc/oc/pgm/tablist/MatchFooterTabEntry.java]:
```suggestion
      content.append(" - ", TextColor.DARK_GRAY).append(rightContent.colorIfAbsent(TextColor.WHITE));
```

Comment (applenick) [reply core/src/main/java/tc/oc/pgm/tablist/MatchFooterTabEntry.java]: Oh wow whoops 😅

**Comment by Electroid** [comment]
While customizing the footer is nice, I'm not sure having the left/right split makes much sense (other than for legacy reasons). It's kinda odd, but I'm not sure of a different solution.

**Comment by Pablete1234** [comment]
I think just having right/left is the easiest way of just letting any generic text in there.
The alternative would be starting to get into what shows and what doesn't show (as in, allowing to remove things like the time) but i think that's a completely different topic, and this one is simple and useful enough

**Comment by applenick** [comment]
I’d also like to point out we do allow similar customization of the sidebar 🤷‍♂️ 

I decided to go with left/right, as @Pablete1234 mentioned, it’s the easiest way of implementing this feature without getting into more complex integrations. 

It’s likely in our favor to offer some some form of this feature and perhaps additional tablist design choices going forward. I imagine that will be an appreciated feature once PGM is in the hands of more general server owners who want to brand their servers further.

Let me know if you have any suggestions on the config or alternatives. I’d be happy to adjust this however you see fit 👍

**Review by Electroid — APPROVED** [review]

**Pull request merged**